ground/src/main/java/com/google/android/ground/ui/map/gms/mog/MogSource.kt

```diff
@@ -17,20 +17,27 @@
 package com.google.android.ground.ui.map.gms.mog
 
 /**
- * Provides URLs of one or more MOGs partitioned by web mercator tile extents at a particular range
- * of zoom level.
+ * Provides URLs or relative paths of one or more MOGs partitioned by web mercator tile extents at a particular range
+ * of zoom level. Relative paths are assumed to be managed by RemoteStorageManager.
  *
  * Examples:
  * ```
  *   // Tiles for zoom levels 0-7 (inclusive) contained in a single file:
- *   val world = MogSource("https://storage.googleapis.com/my-bucket/world.tif", 0..7)
+ *   val world = MogSource(0..7, "https://storage.googleapis.com/my-bucket/world.tif")
+ *   or
```

Comment:
Looks like the wrapping got munged a bit for some reason.

Reply:
This was intentional because I didn't want the `or` to be included inadvertently if someone copies and paste. Just removing the `or` part.